live/provision-stacker.yaml

```diff
@@ -0,0 +1,58 @@
+provision-base:
+  build_only: true
+  from:
+    type: docker
+    url: "docker://zothub.io/machine/bootkit/rootfs:0.0.5.230327-squashfs"
+
+provision-rootfs-pkg:
+  build_only: true
+  from:
+    type: built
+    tag: provision-base
+  run: |
+    pkgtool install udev kmod \
+        libsquashfs-dev tpm2-tools cryptsetup
+
+provision-rootfs:
+  from:
+    type: built
+    tag: provision-rootfs-pkg
+  import:
+    - https://github.com/project-machine/mos/releases/download/0.0.9/mosctl
+    - trust-provision
+    - trust-provision.service
+    - trust-provision-failed.service
+    - console-helper
+  run: |
+    #!/bin/sh -ex
+    writefile() {
+      mkdir -p "${1%/*}"
+      echo "write $1" 1>&2
+      cat >"$1"
+    }
+
+    writefile /etc/systemd/network/20-wire-enp0s-dhcp.network <<"END"
+    [Match]
+    Name=enp0s*
+    [Network]
+    DHCP=yes
+    END
+
+    cd /stacker
+    cp mosctl trust-provision console-helper /usr/bin
+    ( cd /usr/bin && chmod 755 mosctl trust-provision console-helper )
+
+    cp trust-provision.service trust-provision-failed.service \
+        /etc/systemd/system/
+
+    cd /
+    mkdir -p /etc/systemd/system/multi-user.target.wants
+
+    cd /etc/systemd/system/
+    for s in trust-provision*.service; do
+      ln -s "$PWD/$s" "/etc/systemd/system/multi-user.target.wants/$s"
+    done
```

Comment:
any reason not to systemctl enable trust-provision.service trust-provision-failed.service ?

Reply:
I don't know - I copied this from Scott's better-trust-running pr.  I'm actually not sure what the PWD is meant to be...  You wouldn't think that would work as is.  (But it does.  provisioning did succeed)